src/Microsoft.ApplicationInsights.AspNetCore/ExceptionTrackingMiddleware.cs

```diff
@@ -1,42 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
-{
-    using System;
-    using System.Threading.Tasks;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-    using Microsoft.AspNetCore.Http;
-
-    /// <summary>
-    /// Sends telemetry about exceptions thrown by the application to the Microsoft Application Insights service.
-    /// </summary>
-    public sealed class ExceptionTrackingMiddleware
```

Comment:
Just to be clear I understand, that will get reported to AI?  Can you give me a quick tip on what the path is, once everything is wired up, so that this data gets sent to AI when handled in the framework (since the framework doesn't know about AI as such)?